curvefs/src/client/s3/client_s3_adaptor.cpp

```diff
@@ -97,15 +113,15 @@ int S3ClientAdaptorImpl::Write(uint64_t inodeId, uint64_t offset,
     int64_t exceedRatio = memCacheRatio - memCacheNearfullRatio_;
     if (exceedRatio > 0) {
         // upload to s3 derectly or cache disk full
-        if (!enableDiskCache_ ||
-            (enableDiskCache_ && diskCacheManagerImpl_->IsDiskCacheFull())) {
-            uint32_t exponent = pow(2, (exceedRatio) / 10);
-            bthread_usleep(throttleBaseSleepUs_ * exceedRatio * exponent);
+        if ((diskCacheType_ < DiskCacheType::ReadWrite) ||
+             ((diskCacheType_ >= DiskCacheType::ReadWrite) &&
```

Comment:
what does this mean: 
diskCacheType_ < DiskCacheType::ReadWrite) ||
             ((diskCacheType_ >= DiskCacheType::ReadWrite

Reply:
if disk write cache is not enable，and the memory cache if full，we should sleep a little time to wait for data upload to s3，otherwise the latency of user io will jitter seriously， because the io latency of memory cache is very short than s3 backend.